src/AltinnCore/Common/Configuration/ServiceRepositorySettings.cs

```diff
@@ -389,6 +399,18 @@ public string GetDynamicsPath(string org, string service, string developer)
             return GetServicePath(org, service, developer) + RESOURCE_FOLDER_NAME + DYNAMICS_FOLDER_NAME;
         }
 
+        /// <summary>
+        /// Gets the full path to Calculation
+        /// </summary>
+        /// <param name="org">The Organization code for the service owner</param>
+        /// <param name="service">The service code for the current service</param>
+        /// <param name="developer">the developer for the current service</param>
+        /// <returns>The full path, ending with "/"</returns>
```

Comment:
"The full path to the calculation folder, ending with '/'"?

Reply:
fixed